plugins/maciVoting/pages/proposal.tsx

```diff
@@ -55,7 +52,7 @@ export default function ProposalDetail({ id: proposalId }: { id: string }) {
         <div className="mg:gap-y-6 flex w-full flex-col gap-6 md:flex-row md:gap-x-12">
           <div className="flex flex-col gap-y-6 md:w-[63%] md:shrink-0">
             <BodySection body={proposalMetadata?.description || "No description was provided"} />
-            <If condition={canFinalize && finalizeStatus !== "submitted"}>
```

Comment:
In this case I realized that we do not need to check `finalizeStatus` different than `submitted` because if the results have been submitted by the coordinator, `isTallied` will return true and therefore `canFinalize` will return false.

I also removed it because of the problem of having the same `finalizeStatus` for all polls (which was fixed by using the `useCoordinatorFinalize` hook)